### DISCUSSION TITLE
Disable rounded corner option does not stay selected - Known bug or user error?

### DISCUSSION THREAD

**Discussion by Tritan20** [valinet/ExplorerPatcher#4082] [Q&A]
After reinstalling EP after the new Windows update, I wanted to revert back to nice clean corners. The only issue is, after windows prompts you if you wish to continue, nothing changes and the setting for disabling rounded corners is no longer checked. Is this a known bug or user error?

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3555

please close this.
